docs/src/core/transaction-request/MERCHANT_INTEGRATION.md

```diff
@@ -0,0 +1,270 @@
+
+```javascript
+const index = async (request, response) => {
+    if (request.method === 'GET') return get(request, response);
+    if (request.method === 'POST') return post(request, response);
+
+    throw new Error(`Unexpected method ${request.method}`);
+};
+```
+
+We set up our handler to only respond to `GET` and `POST` requests.
+
+```javascript
+if (request.method === 'GET') return get(request, response);
+if (request.method === 'POST') return post(request, response);
+```
+
+## 2. The link
+
+A Solana Pay transaction request URL describes an interactive request for any Solana transaction.
+
+```html
+solana:<link>
+```
+
+A single [link][3] field is required as the pathname. The value must be an absolute HTTPS URL. If the URL contains query parameters, it must be URL-encoded.
+
+```html
+solana:https://example.solanapay.com
+```
+
+Our server `https://example.solanapay.com` needs to be configured to respond correctly to `GET` and `POST` requests.
+
+## 3. The GET request
+
+The first part of the transaction request spec is for the wallet to make a `GET` request to the specified link.
+
+```javascript
+const get = async (request, response) => {
+    const label = 'Exiled Apes Academy';
+    const icon = 'https://exiledapes.academy/wp-content/uploads/2021/09/X_share.png';
+
+    response.status(200).send({
+        label,
+        icon,
+    });
+};
+```
+
+The `GET` endpoint should respond with two properties. `label` describes the source of the transaction request. For example, this might be the name of a brand, store, application, or person making the request. `icon` must be an SVG, PNG, or WebP image. The icon and label will be displayed to the user.
+
+## 4. The POST request
+
+The second part of the transaction request spec is the `POST` request.
+
+```javascript
+import {clusterApiUrl, Connection, Keypair, PublicKey, Transaction} from '@solana/web3.js';
+import BigNumber from 'bignumber.js';
+import {createTransferCheckedInstruction, getAccount, getAssociatedTokenAddress, getMint} from "@solana/spl-token";
+import {TEN} from "@solana/pay";
+
+const splToken = new PublicKey(process.env.USDC_MINT);
+const MERCHANT_WALLET = new PublicKey(process.env.MERCHANT_WALLET);
+
+const post = async (request, response) => {
+    // Account provided in the transaction request body by the wallet.
+    const accountField = request.body?.account;
+    if (!accountField) throw new Error('missing account');
+    
+    const sender = new PublicKey(accountField);
+
+    try {
+
+        // create spl transfer instrution
+        const splTransferIx = await createSplTransferIx(sender, connection);
+
+        // create the transaction
+        const transaction = new Transaction();
+
+        // add the instruction to the transaction
+        transaction.add(splTransferIx);
+
+        // Serialize and return the unsigned transaction.
+        const serializedTransaction = transaction.serialize({
+            verifySignatures: false,
```

Comment:
Nit: this doesn't need to be false in my experience. Probably worth keeping it true so that if the TX is partially signed then those signatures that are present are validated.

Reply:
Hmm. Why would we need to verify our own signature?

Reply:
I don't think it's super important, but in theory I guess the server could be requesting other signatures by some channel before returning the transaction or there could be some issue with a signature, since there aren't any restrictions on what the transaction could be that's generated. It's definitely a nit, but IIRC the default is true and I don't think there's a case where making it false is necessary. If there was some issue with a signature of the returned transaction and we use false then it'll presumably error when the wallet tries to sign/send the transaction, so I think it'd generally be better to have it error here instead